addons/web/static/src/js/views/search_panel.js

```diff
@@ -231,19 +231,21 @@ const SearchPanel = Widget.extend({
     async update(params) {
         const currentDomain = JSON.stringify([...this.searchDomain, ...this.viewDomain]);
         const newDomain = JSON.stringify([...params.searchDomain, ...params.viewDomain]);
-        if (this.needReload || (currentDomain !== newDomain)) {
-            this.needReload = false;
+        if (this.reloadOptions || (currentDomain !== newDomain)) {
```

Comment:
weird to use an 'options' dict as flag

Reply:
also, should be initialized in init

Reply:
we keep things like that but reloadOptions has been initialized in init